### PR TITLE
feat(auto-complete): add focus and blur method

### PR DESCRIPTION
**Pull request by LadyChatterleyLover**
## Types of changes

- [x] New feature
- [ ] Bug fix
- [ ] Enhancement
- [ ] Component style change
- [ ] Typescript definition change
- [ ] Documentation change
- [ ] Coding style change
- [ ] Refactoring
- [ ] Test cases
- [ ] Continuous integration
- [ ] Breaking change
- [ ] Others

## Background and context

for issue https://github.com/arco-design/arco-design-vue/issues/1749
## Solution

 add focus and blur method
## Changelog

| Component | Changelog(CN) | Changelog(EN) | Related issues |
| --------- | ------------- | ------------- | -------------- |
|           |               |               |                |

## Checklist:

- [x] Test suite passes (`npm run test`)
- [x] Provide changelog for relevant changes (e.g. bug fixes and new features) if applicable.
- [x] Changes are submitted to the appropriate branch (e.g. features should be submitted to `feature` branch and others
  should be submitted to `main` branch)

### CODE REVIEW

**Review by hehehai — COMMENTED** [review]
麻烦分开提交吧，头像图片这个或许有 break

File: packages/web-vue/components/auto-complete/auto-complete.tsx [hunk 1]
```diff
@@ -283,6 +283,17 @@ export default defineComponent({
       );
     };
 
+    const inputBlur = () => {
```
Comment (hehehai) [inline packages/web-vue/components/auto-complete/auto-complete.tsx:286]: 这里用 methods 的方式吧，方便生成文档

**Comment by LadyChatterleyLover** [comment]
晕，忘记切分支了，马上改一下




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "arco-design/arco-design-vue"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月31日(星期一) 下午2:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [arco-design/arco-design-vue] feat(auto-complete): add focus and blur method (PR #1808)





 
@hehehai commented on this pull request.
 
麻烦分开提交吧，头像图片这个或许有 break

**Comment by LadyChatterleyLover** [comment]
好，




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "arco-design/arco-design-vue"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月31日(星期一) 下午2:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [arco-design/arco-design-vue] feat(auto-complete): add focus and blur method (PR #1808)





 
@hehehai commented on this pull request.
 
 
In packages/web-vue/components/auto-complete/auto-complete.tsx:
 &gt; @@ -283,6 +283,17 @@ export default defineComponent({        );      };   +    const inputBlur = () =&gt; {  
这里用 methods 的方式吧，方便生成文档

**Pull request closed without merging**